wire-ios-request-strategy/Sources/Helpers/EventDecoderDecryptionTests.swift

```diff
@@ -141,104 +136,4 @@ final class EventDecoderDecryptionTests: MessagingTestBase {
         }
     }
 
-    func testThatItInsertsAnUnableToDecryptMessageIfTheEncryptedPayloadIsLongerThan_18_000() async throws {
```

Comment:
Deleted these tests because we don't do the validation of the cipher text anymore.